.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+*                             @LanikSJ
+README.md                     @krazyito65 @Zweih
+dark-theme.css                @krazyito65 @Zweih
+event-listener.js             @krazyito65 @Zweih
+slack-dark-mode.ps1           @niowe @BabylonZeus @krazyito65
```

Comment:
Pretty sure this is supposed to be `@nlowe` not `@niowe`

Reply:
😭